assets/vue/components/documents/DocumentEntry.vue

```diff
@@ -52,6 +52,9 @@ const dataType = computed(() => {
   if (isVideo(props.data)) {
     return 'video'
   }
+  if (isAudio(props.data)) {
+    return 'video'
+  }
```

Comment:
This property is set in :data-type="dataType". I couldn't find anything in documentation about audio. I couldn't use iframe and set the height of controls either. This approach works but show controls like a video. 

It's not ideal but at least user can see the audio controls.